lib/core/src/lsUtil.cpp

```diff
@@ -407,6 +406,12 @@ lsCollUtil( rcComm_t *conn, rodsPath_t *srcPath, rodsEnv *myRodsEnv,
         printCollInheritance( conn, srcColl );
     }
 
+    // Return immediately if the user passed the "-d" option.
+    // This option allows "ils" to mirror the behavior of "ls -d <directory>".
+    if (True == rodsArgs->dataObjects) {
```

Comment:
dataObjects?    rather than 'directory listing' or something?

Reply:
That is the variable name that is assigned to the -d option. I can't change it. No one will see that though.